src/core/utils/transactions.ts

```diff
@@ -175,20 +175,18 @@ const getAssetFromChanges = (
   return changes[0]?.asset;
 };
 
-const getAddressTo = (
-  meta: PaginatedTransactionsApiResponse['meta'],
-  changes: {
-    direction: TransactionDirection;
-    asset: ParsedUserAsset;
-    address_to: Address;
-  }[],
-) => {
-  if (meta.type === 'approve') {
-    return meta?.approval_to;
+const getAddressTo = (tx: PaginatedTransactionsApiResponse) => {
```

Comment:
not sure what these `meta.type`s are, but should there be a case for `swap` instead of just defaulting?